controllers/bsl_test.go

```diff
@@ -118,6 +549,8 @@ func TestVeleroReconciler_ValidateBackupStorageLocations(t *testing.T) {
 				EventRecorder: record.NewFakeRecorder(10),
 			}
 			got, err := r.ValidateBackupStorageLocations(r.Log)
+			fmt.Println(fmt.Sprintf("ValidateBackupStorageLocations() error = %v, wantErr %v", err, tt.wantErr))
```

Comment:
NIT: can we just use `fmt.Printf`? 

Also, do we we even need these prints? `t.Errorf` handles this message for us already i think